core/src/main/java/com/linecorp/armeria/server/ServiceConfigSetters.java

```diff
@@ -157,4 +159,23 @@ ServiceConfigSetters blockingTaskExecutor(ScheduledExecutorService blockingTaskE
      */
     @UnstableApi
     ServiceConfigSetters multipartUploadsLocation(Path multipartUploadsLocation);
+
+     /**
+      * Sets a {@link EventLoopGroup worker group} to be used when executing service codes.
+      *
+      * @param blockingTaskExecutor the {@link ScheduledExecutorService executor} to be used.
+      * @param shutdownOnStop whether to shut down the {@link ScheduledExecutorService} when the {@link Server}
+      *                       stops.
+      */
+     ServiceConfigSetters serviceWorkerGroup(EventLoopGroup serviceWorkerGroup,
+                                             boolean shutdownOnStop);
```

Comment:
- Add `@UnstableApi`
- Indent?
- The `@param` Javadoc is different from the actual parameters.